modin/utils.py

```diff
@@ -527,6 +527,42 @@ def try_cast_to_pandas(obj: Any, squeeze: bool = False) -> Any:
     return obj
 
 
+def repartition(df, axis: Optional[int] = None):  # type:ignore
+    """
+    Repartitioning Modin objects to get ideal partitions inside.
+
+    Allows to improve performance where the query compiler cannot yet.
+
+    Parameters
+    ----------
+    df : DataFrame or Series
+    axis : int, optional
+
+    Returns
+    -------
+    DataFrame or Series
+        The repartitioned dataframe or series, depending on the original type.
+    """
+    if axis not in (0, 1, None):
+        raise NotImplementedError
+
+    from modin.pandas import DataFrame, Series
+
+    if not isinstance(df, (DataFrame, Series)):
+        raise NotImplementedError
+
+    list_axis = [0, 1] if axis is None else [axis]
```

Comment:
tuples tend to be more efficient (as they're immutable)
```suggestion
    list_axis = (0, 1) if axis is None else (axis,)
```